content/hi/docs/setup/learning-environment/_index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: सीखने का माहौल
+weight: 20
+---
+
+<!--
+{{/* There is a Netlify redirect from this page to /docs/tasks/tools/ */}}
+{{/* This page content only exists to provide a navigation stub */}}
+{{/* and to protect in case that redirect is one day removed. */}}
+
+{{/* If you're localizing this page, you only need to copy the front matter */}}
+{{/* and add a redirect into "/static/_redirects", for YOUR localization. */}}
+-->
+
+## kind
+
+[`kind`](https://kind.sigs.k8s.io/docs/) आपको Kकुबेरनेट्स को अपने स्थानीय कंप्यूटर पर चलाने देता है। इहै।इस
+ उपकरण के लिए आवश्यक है कि आपके पास 
+[Docker](https://docs.docker.com/get-docker/)स्थापित और कॉन्फ़िगर किया गकिया गया हो।
+
```

Comment:
1. Kकुबेरनेट्स typo can be fixed.
2. another fix can be made for `इहै।इस` 
3. `गकिया` seems like an extra word here in the sentence